driver/config/provider.go

```diff
@@ -277,9 +277,6 @@ func (p *DefaultProvider) CookieSameSiteMode(ctx context.Context) http.SameSite
 	case "strict":
 		return http.SameSiteStrictMode
 	case "none":
-		if p.IsDevelopmentMode(ctx) {
```

Comment:
Please revert this - but you can add that:

1. If TLS is disabled and dev mode is on, (you can check this via the issuer or public URL for example) then set this to lax
2. If TLS is enabled then return "None"

Reply:
This won't work for Ory network, as TLS is always terminated before reaching Hydra there.

Reply:
We can identify whether TLS is enabled by looking at the public or issuer URL. If that is prefixed https, we're serving TLS.

Reply:
pure genius :)